sandboxes/dynamicanalysis/Dockerfile

```diff
@@ -0,0 +1,129 @@
+# Debian can't be used because the public keys for the powershell and kubectl repos
+# aren't recognised. Installation fails with 'The following signatures couldn't be
+# verified because the public key is not available'
+# NO_PUBKEY EB3E94ADBE1229CF [powershell]
+# NO_PUBKEY B53DC80D13EDEF05 [kubectl]
+
+# buildpack-deps 22.04 [ubuntu]
+FROM buildpack-deps@sha256:d76488a56e1e85930e053b794daa9f990636565bf0de54903d65d93c43c3cad5 AS image
+
+ENV DEBIAN_FRONTEND noninteractive
+# Silence 'debconf: delaying package configuation, since apt-utils is not installed' warning
+# as it related to interactive configuration anyway
+ENV DEBCONF_NOWARNINGS="yes"
+
+# Essentials
+RUN apt-get update && apt-get -y upgrade && apt-get install -y --no-install-recommends \
```

Comment:
Can we move these down to the places where the language specific scripts are being copied? 

That way it's clearer which dependencies are needed where, and we keep all the language/ecosystem specific pieces grouped in a logical way.

Reply:
Done